databricks/koalas/series.py

```diff
@@ -4730,6 +4730,31 @@ def asof(self, where):
         result_series.name = self.name
         return result_series
 
+    def mad(self):
+        """
+        Return the mean absolute deviation of values.
+
+        Examples
+        --------
+        >>> s = ks.Series([1, 2, 3, 4])
+        >>> s
+        0    1
+        1    2
+        2    3
+        3    4
+        Name: 0, dtype: int64
+
+        >>> s.mad()
+        1.0
+        """
+
+        sdf = self._internal.spark_frame
+        column_name = self._internal.data_spark_column_names[0]
+        avg = _unpack_scalar(sdf.select(F.avg(sdf[column_name])))
+        mad = _unpack_scalar(sdf.select(F.avg(F.abs(sdf[column_name] - avg))))
```

Comment:
nit:
```diff
-        column_name = self._internal.data_spark_column_names[0]
-        avg = _unpack_scalar(sdf.select(F.avg(sdf[column_name])))
-        mad = _unpack_scalar(sdf.select(F.avg(F.abs(sdf[column_name] - avg))))
-
+        spark_column = self._internal.spark_column
+        avg = _unpack_scalar(sdf.select(F.avg(spark_column)))
+        mad = _unpack_scalar(sdf.select(F.avg(F.abs(spark_column - avg))))
```

LGTM otherwise.